tests/scale/threaded_utils/virt.py

```
@@ -0,0 +1,240 @@
+def verify_guest_data(data_before_action: dict, data_after_action: dict) -> None:  # skip-unused-code
+    """
+    Verify data gathered using GUEST_DATA_COMMAND_LIST before and after an event or action
+
+    Args:
+        data_before_action (dict): Data before event or action
+        data_after_action (dict): Data after event or action
+    """
+    return_errors = []
+
+    if not (data_before_action and data_after_action and data_before_action != data_after_action):
+        raise ValueError(f"invalid input: before:{data_before_action!r} after:{data_after_action!r}")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)